x-pack/plugins/observability/public/components/alerts_table/register_alerts_table_configuration.tsx

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { AlertTableConfigRegistry } from '@kbn/triggers-actions-ui-plugin/public/application/alert_table_config_registry';
+import { ObservabilityRuleTypeRegistry } from '../..';
+import { getAlertsPageTableConfiguration } from './alerts/get_alerts_page_table_configuration';
+import { getRuleDetailsTableConfiguration } from './rule_details/get_rule_details_table_configuration';
+import { getSloAlertsTableConfiguration } from './slo/get_slo_alerts_table_configuration';
+
+export const registerAlertsTableConfiguration = (
+  alertTableConfigRegistry: AlertTableConfigRegistry,
+  observabilityRuleTypeRegistry: ObservabilityRuleTypeRegistry,
+  config: any
```

Comment:
@maryam-saeidi Can we have a type other than `any`?

Reply:
Good catch, fixed!